web/modules/custom/joinup_user/joinup_user.module

```diff
@@ -329,8 +441,42 @@ function joinup_user_entity_base_field_info(EntityTypeInterface $entity_type) {
  * Implements hook_ENTITY_TYPE_presave() for OG Membership entities.
  */
 function joinup_user_og_membership_presave(EntityInterface $entity) {
-  // When a membership changes, clear the user's cache so that the related
-  // content on the user profile page matches the new group memberships.
+  joinup_user_invalidate_user_cache_tags($entity);
```

Comment:
Why is this documentation removed? It explains why it is needed to invalidate the cache tags. Without this documentation this line becomes really suspicious.